demo/build.gradle

```diff
@@ -1,27 +1,36 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.2"
+    compileSdkVersion 33
+    buildToolsVersion "25.0.3"
     defaultConfig {
         applicationId "com.flurgle.camerakit.demo"
-        minSdkVersion 15
-        targetSdkVersion 25
+        minSdkVersion 21
+        targetSdkVersion 33
         versionCode 4
         versionName "1.0.3"
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+
+        multiDexEnabled true
```

Comment:
Are we using multidex? 😱

Reply:
Let me double check this.

Reply:
It was not needed for the camerakit - I've tested and removed.